runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -792,10 +885,11 @@ private void applyForEachAddress(BiConsumer<Long, ILogData> logDataProcessor) {
         long startAddress;
         boolean ended = false;
         boolean started = false;
+        boolean cont = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651460)